src/components/search/index.tsx

```diff
@@ -34,6 +34,7 @@ type SearchProps = {
   searchClient?: any
   searchState?: any
   instructor?: any
```

Comment:
this stands out to me. I do t like “sanity” as special data. could we merge this into a si file “instructor” object so it’s “just an instructor” and then display data based on the shape of instructor and the available properties